packages/node_modules/@ciscospark/plugin-messages/src/messages.js

```diff
@@ -95,14 +107,34 @@ const Messages = SparkPlugin.extend({
    * @param {Object} activity from mercury
    * @returns {Object} constructed event
    */
-  createMessagesEventData(activity) {
+  messageCreatedEventData(activity) {
```

Comment:
Why rename? It actually is creating a new object of `messages` data.

Reply:
I wanted to differentiate between the event for the message created event and the message deleted event, but I from your question can see that the create part of the name refers to what the function is doing (not the contents of the event its creating).

What about createMessageCreatedEventData and createMessageDeletedEventData?  (or a single createMessageEventData, with a second activity parameter)?

Reply:
We're wading into Java naming territory 😉 

Let's simplify them to these, because we're already within the "messages" plugin and file: 
- `getCreatedEvent`
- `getDeletedEvent`

Reply:
Will do!